LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Controllers/SearchController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LvivCompany.Bookstore.DataAccess.Repo;
+using LvivCompany.Bookstore.Entities;
+using LvivCompany.Bookstore.Web.Mapper;
+using LvivCompany.Bookstore.Web.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace LvivCompany.Bookstore.Web.Controllers
+{
+    public class SearchController : Controller
+    {
+        private IRepo<Book> _bookRepo;
+        private IMapper<Book, BookViewModel> _bookmapper;
+
+        public SearchController(IRepo<Book> bookRepo, IMapper<Book, BookViewModel> bookmapper)
+        {
+            _bookRepo = bookRepo;
+            _bookmapper = bookmapper;
+        }
+
+        public async Task<IActionResult> Index(string SearchText)
+        {
+            List<Book> books = (await _bookRepo.GetAllAsync()).ToList();
```

Comment:
1. Try to refactor codebase so that `(await _bookRepo.GetAllAsync()).ToList()` won't appear everywhere
2. Don't query all books from database. It's not optimal
3. If term is not found, return `no results found`